content/embeds/tryout-redisearch-2.md

```diff
@@ -0,0 +1,93 @@
+## Creating indexes
+
+Let's create a new index called "myIdx". When you define the index, you
+must pass in the structure of the data you are adding to the index.
+In this example, we have four things we input: title, body, url, and
+value. In this example, we have three TEXT and one NUMERIC values. The
+title has a weight of 5.0.
```

Comment:
Let's create a new index called "myidx". When defining an index, one should
pass in the structure of the data being indexed.
In this example, we have four things we index: the title, body, url, and value.
We have three TEXT values and one NUMERIC. The title has a weight of 5.0.

Reply:
- Please use a concrete index name. "myidx" is too abstract and generic. "post_idx" might make more sense in this case.
- We're moving away from the impersonal "one" in favor of "you".
- s/four things/four fields/